config/config.yaml

```
@@ -249,3 +249,6 @@ PgNotifier:
   retriggerCount: 500
   trackBatchInterval: 2s
   maxAttempt: 3
+Reporting:
+  eventSamplingEnabled: false
+  eventSamplingDurationInMinutes: 60
```

Let's use more hierarchical config ?
```suggestion
Reporting:
  eventSampling:
      enabled: false
      durationInMinutes: 60
```

---

actually, no need to add these in this file, since we are using reloadable configs